src/libs/API.js

```diff
@@ -19,7 +19,14 @@ function write(command, apiCommandParameters = {}, onyxData = {}) {
     // Assemble all the request data we'll be storing in the queue
     const request = {
         command,
-        data,
+        data: {
+            ...data,
+
+            // We add this here because this was passed in the old api requests and some middlewares still expect this.
```

Comment:
NAB, this comment is not necessary. Whoever wants to remove this can just look to see if it's being used and why.